recipes/nuraft/all/conandata.yml

```diff
@@ -0,0 +1,16 @@
+sources:
+  "cci.20220801":
```

Comment:
This project has releases https://github.com/eBay/NuRaft/releases is there a reason to use master?

Reply:
There has not been a release for over a year even though there has been a lot of fixes that are worthwhile to pick up. Author also doesn't use semantic versioning anyways between release numbers, so would likely mis-match with what we call it in the recipe.

Reply:
Would you mind asking in the upstream repository for a new release? It is always better if we can use tagged released.

Reply:
I don't mind, but I know historically they have been using X.Y.Z style tags without adhering semantic versioning rules. I'll check what they want to do (cut a new release with non-semantic tag style, or to start adhering to the guidelines). :+1:

Reply:
We do not require semver, we just prefer official releases (they can do anything they'd like for that)